client/actions/actions.init.js

```diff
@@ -0,0 +1,34 @@
+import { ActionType } from 'redux-promise-middleware';
+
+export const ActionsInit = 'ActionsInit';
+export const INIT_FULFILLED = `${ActionsInit}_${ActionType.Fulfilled}`;
+
+export function init() {
+  const payload = new Promise((resolve) => {
+    const data = {};
+    const requests = ['categories', 'orders', 'filters', 'discounts'];
```

Comment:
@IgorChaikin Lets get rid of this `actions.init.js` file, and move actions for fetching `categories`, `filters`, `discount` to `actions.categories.js`, `filters.categories.js` and `actions.discounts.js` and run them when `App` container is mounted